puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-documentation.yaml

```diff
@@ -0,0 +1,9 @@
+- job:
+    name: foreman-documentation-test
+    node: 'admin && sshkey'
+    scm:
+      - git:
+          url: "https://github.com/theforeman/foreman-documentation"
+          wipe-workspace: false
```

Comment:
Just keeping the workspace doesn't mean you archive any result. It just means that you *may* get a non-clean workspace from a previous job when a new one starts.